README.md

```diff
@@ -308,17 +318,60 @@ To fix formatting errors, run the following:
 ```sh
 yarn validate:eslint --fix
 ```
-
 Remember to cover your changes with tests if possible.
 
+# Running the example app 
+While developing, you can run the example app to test your changes.
+
+## Setup
+
+- Clone the repository 
+- Run `yarn` in the root directory to install dependencies, and again in `src` to create the dist build.
+- (on iOS) Run `npx pod-install` from the `example` directory
+- (on Windows) You need to manually link the Slider module to your project.
+
+## Start the app
+
+- Run `yarn run:android` to run on Android
+- Run `yarn run:ios` to run on iOS
+- Run `yarn run:web` to run on web
+
+### Windows
+- Run `yarn run:windows` to run the bundler. Then, run the example app solution in Visual Studio.
+
```

Comment:
can we simplify this? yarn run:windows should be able to do npx react-native run-windows?

Reply:
hmm, that might be a good idea. I just followed what we put in the docs